Ignore the exception detail for two exceptions that are slightly
different on pypy.

diff -r 7f90a96378ed tests/test_template.txt
--- a/tests/test_template.txt	Mon Mar 05 21:14:08 2012 +0100
+++ b/tests/test_template.txt	Wed May 16 23:29:46 2012 +0200
@@ -6,7 +6,7 @@
     'Hi Ian'
     >>> Template('Hi {{repr(name)}}').substitute(name='Ian')
     "Hi 'Ian'"
-    >>> Template('Hi {{name+1}}').substitute(name='Ian')
+    >>> Template('Hi {{name+1}}').substitute(name='Ian') #doctest: +IGNORE_EXCEPTION_DETAIL
     Traceback (most recent call last):
         ...
     TypeError: cannot concatenate 'str' and 'int' objects at line 1 column 6
@@ -125,7 +125,7 @@
     >>> sub('{{default x=1}}{{x}}')
     '1'
     >>> # The normal case:
-    >>> sub('{{x}}')
+    >>> sub('{{x}}') #doctest: +IGNORE_EXCEPTION_DETAIL
     Traceback (most recent call last):
         ...
     NameError: name 'x' is not defined at line 1 column 3
